app/rdf/query-search.ts

```diff
@@ -147,12 +143,21 @@ export const searchCubes = async ({
         ?iri ${ns.schema.datePublished} ?datePublished .
         FILTER(DATATYPE(?datePublished) = ${ns.xsd.date})
       }
+
       OPTIONAL {
         ?iri ${ns.dcat.theme} ?themeIri .
-        ${buildLocalizedSubQuery("themeIri", "schema:name", "themeLabel", {
-          locale,
-        })}
+        GRAPH <https://lindas.admin.ch/sfa/opendataswiss> {
+          ?themeIri a ${ns.schema.DefinedTerm} ;
+          ${
+            ns.schema.inDefinedTermSet
+          } <https://register.ld.admin.ch/opendataswiss/category> .
+          ${buildLocalizedSubQuery("themeIri", "schema:name", "themeLabel", {
+            locale,
+          })}
+        }
```

Comment:
Curiosity : Why do we need another graph for themes ?

Reply:
I think it's better for the performance if we narrow down the graph, so it doesn't need to look everywhere to find the things? @Rdataflow might have a better understanding here :)

Reply:
@bprusinowski well understood 😄 it's about narrowing down to one `GRAPH` (a.k.a. one partition of the SPARQL database)